PubnativeLite/PubnativeLite/HyBidSettings.m

```diff
@@ -113,6 +113,7 @@ - (NSString *)orientation {
 }
 
 - (NSString *)deviceSound {
+    [[AVAudioSession sharedInstance] setActive:YES error:nil];
```

Comment:
In the past setting the `AVAudioSession` active caused an issue where movie/audioplayer stoped for the user. We made a hot fix in order to fix the issue. You can check the issue here: https://github.com/pubnative/pubnative-hybid-ios-sdk/issues/228

and the hotfix commit sha here: a8def0c32de8f1e397492617eab549ccd51ee434

Not adding the `AVAudioSession` still works based on my experiments in the past, which still gives us a soldi information whether the device muted or not. So if your integration works without this code, please remove it.

Reply:
@can-soykarafakili but without that line it does not work. At least on the simulator. It started to work only after adding that line, otherwise it was returning the same value for both modes

Reply:
@can-soykarafakili I've tested about 20-30 times with and without that line and it did not work reliably without that line. So we have to think about the solution then 🤔

Reply:
@orkhanalizada2020  Try with the a device. I've done tests back then with my device and it was working fine. Volume Up & Down buttons registered fine in device tests. (Didn't remember the mute switch) I was able to detect whether the device was muted or not.